topics/index.md

```diff
@@ -0,0 +1,8 @@
+---
+aliases: lgbtq, lgbt
+display_name: queer
+short_description: Queer and LGBTQ are umbrella terms for sexual orientations, gender identities and gender expressions.
+topic: queer
+wikipedia_url: https://en.wikipedia.org/wiki/Queer
+---
+Queer and LGBTQ are umbrella terms for sexual orientations, gender identities and gender expressions. This includes trans*, inter*, homo-, bi-, pan-, a-sexuality, …. The projects in this category most often try to enable minorites and fight discrimination.
```

Comment:
Because this file is interpreted as Markdown, the "\*"s here will be parsed as italics rather than literal asterisk characters. Let's escape them so they show up properly:

```suggestion
Queer and LGBTQ are umbrella terms for sexual orientations, gender identities and gender expressions. This includes trans\*, inter\*, homo-, bi-, pan-, a-sexuality, …. The projects in this category most often try to enable minorites and fight discrimination.
```

(... Huh. I notice they show up okay already on [the "View File" page](https://github.com/github/explore/blob/ab61ab57bcaaa3aec18df41ceabea4b3c152ec4a/topics/index.md)? Let's do the escaping anyway for safety - I'm not 100% certain how consistent the rendering is between that view and the actual Topic page 🤔 )